platform/golden/bullseye.pkr.hcl

```diff
@@ -76,37 +90,36 @@ source "virtualbox-iso" "base-debian-amd64" {
   skip_nat_mapping     = false
   iso_checksum         = "${var.iso_checksum_type}:${var.iso_checksum}"
   iso_url              = "${var.iso_url}"
-  iso_interface        = "virtio"
+  iso_interface        = "sata"
   nic_type             = "virtio"
   hard_drive_interface = "virtio"
   rtc_time_base        = "UTC"
   shutdown_command     = "sudo -S shutdown -P now"
   ssh_username         = "packer"
   ssh_password         = "${var.ssh_password}"
   ssh_wait_timeout     = "30m"
+  firmware             = "efi"
+  # don't remove the built VM from VirtualBox after export
+  keep_registered      = "true"
+  # don't export the built VM
+  skip_export          = "false"
   vboxmanage = [
     # enable recording video of install process, for debug and build record
-    [ "modifyvm", "{{.Name}}", "--recording", "on" ],
-    # enable proxy access across NAT network interface.
     [ "modifyvm", "{{.Name}}", "--nat-localhostreachable1", "on" ],
   ]
 }
 
-# Build a golden image by connecting a compute source with a provisioner
-
+## Build a golden image by connecting a compute source with a provisioner
+#
 build {
   sources = ["source.virtualbox-iso.base-debian-amd64"]
-
-# Use Ansible (needs to be on the "packer" host) to provision the instance
   provisioner "ansible" {
     playbook_file = "./provisioners/01_update_packer_user/packer.yml"
-    ansible_env_vars = [
-      "ANSIBLE_HOST_KEY_CHECKING=False",
-      "ANSIBLE_SSH_ARGS='-oForwardAgent=yes -oHostKeyAlgorithms=+ssh-rsa -oPubkeyAcceptedKeyTypes=ssh-rsa'"
+    ansible_ssh_extra_args = [
```

Comment:
line 104, BURRIED in the packer documentation...

The packer docs state ANSIBLE_HOST_KEY_CHECKING=False must be set for the Ansible if using an auto-generated SSH key with packer. We are using an autogenerated key with packer. Going to test this now, if so, I have a lot of image build work that has been, essentially blocked, on a working ssh communicator.

Reference: https://developer.hashicorp.com/packer/integrations/hashicorp/ansible/latest/components/provisioner/ansible
```
ssh_host_key_file (string) - The SSH key that will be used to run the SSH server on the host machine to forward commands to the target machine. Ansible connects to this server and will validate the identity of the server using the system known_hosts. The default behavior is to generate and use a onetime key. Host key checking is disabled via the ANSIBLE_HOST_KEY_CHECKING environment variable if the key is generated.
```